### PR TITLE
Adds AMR Profile status to periodic meter reads E & G

### PR DESCRIPTION
**Pull request by rudidevries**
Connects to SmartSocietyServices/Protocol-Adapter-DLMS#18
Connects to SmartSocietyServices/Webapps#239

### CODE REVIEW

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReads.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.alliander.osgp.dto.valueobjects.smartmetering;
```
Comment (bvdzwet) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReads.java]: This file is missing a license header.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReadsContainer.java [hunk 2]
```diff
@@ -15,20 +15,20 @@
 
     private static final long serialVersionUID = -156966569210717654L;
 
-    private final List<MeterReads> meterReads;
+    private final List<PeriodicMeterReads> meterReads;
     private final PeriodType periodType;
 
-    public PeriodicMeterReadsContainer(final PeriodType periodType, List<MeterReads> meterReads) {
+    public PeriodicMeterReadsContainer(final PeriodType periodType, final List<PeriodicMeterReads> meterReads) {
         this.meterReads = Collections.unmodifiableList(meterReads);
```
Comment (bvdzwet) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReadsContainer.java]: I think that by this way of making the collection immutable, it only is immutable from the perspective of this.meterReads. The collection provided as parameter could still be modified, causing changes to this object.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReadsGas.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package com.alliander.osgp.dto.valueobjects.smartmetering;
```
Comment (bvdzwet) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReadsGas.java]: This file is missing a license header.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReads.java (hunk 1 above)
Comment (rudidevries) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReads.java]: Header was added.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReadsGas.java (hunk 3 above)
Comment (rudidevries) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/PeriodicMeterReadsGas.java]: Header was added.

**Pull request merged**